weekly-mission(week2)/src/view/addErrorMessageToInput.js

```diff
@@ -57,10 +57,13 @@ const addErrorMsgToIncorrectInput = (input) => {
   }
 }
 
-const addErrorMsgToDuplicateInput = (e) => {
+const addErrorMsgToDuplicateInput = async (e) => {
+  if(e.target.value === '') return;
+
   const errorMsg = $('.input-error-msg', e.target.parentNode);
+  const inputEmail = { email: e.target.value };
```

Comment:
요기도 이런 느낌이 좋을 것 같아요!
```suggestion
  const email = { email: e.target.value };
```